README.md

```diff
@@ -56,41 +56,40 @@ Roadmap is [here](https://github.com/ry/deno/blob/master/Roadmap.md).
 
 Also see this presentation: http://tinyclouds.org/jsconf2018.pdf
 
-### Github Noise
-
 I am excited about all the interest in this project. However, do understand that this
 is very much a non-functional prototype. There's a huge amount of heavy lifting to do.
 Unless you are participating in that, please maintain radio silence on github. This
 includes submitting trivial PRs (like improving README build instructions).
 
-## Compile instructions
-
-Get [Depot Tools](http://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up) and make sure it's in your path.
-
-You need [yarn](https://yarnpkg.com/lang/en/docs/install/) installed.
-
-You need [rust](https://www.rust-lang.org/en-US/install.html) installed.
-
-You might want  [ccache](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/ccache) installed.
+## Build instructions
 
-Fetch the third party dependencies.
+To ensure reproducable builds, Deno has most of its dependencies in a git
+submodules.  However, you need
+[rustc](https://www.rust-lang.org/en-US/install.html) installed separately.
 
-    ./tools/build_third_party.py
+You probably want
+[ccache](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/ccache)
+installed too.
 
-Generate ninja files.
+To build:
 
-    gn gen out/Default
-    gn gen out/Release --args='cc_wrapper="ccache" is_official_build=true'
-    gn gen out/Debug --args='cc_wrapper="ccache" is_debug=true '
+    # Fetch deps.
+    git clone --recurse-submodules https://github.com/ry/deno.git
+    cd deno
+    ./tools/run_hooks.py
 
-Then build with ninja (will take a while to complete):
+    # Configure
+    ./third_party/depot_tools/gn gen out/default
+    ./third_party/depot_tools/gn gen out/release --args='cc_wrapper="ccache" is_official_build=true'
+    ./third_party/depot_tools/gn gen out/debug --args='cc_wrapper="ccache" is_debug=true '
 
-    ninja -C out/Debug/ deno
+    # Build
+    ./third_party/depot_tools/ninja -C out/default/ deno
```

Comment:
`./third_party/depot_tools/gn` is not very ergonomic - but that's ok. 
I plan to add a `tools/build.py` which wraps gn and ninja after this patch lands.